app/imports/components/admin/adminIndexLayout.jsx

```diff
@@ -28,6 +28,56 @@ export class AdminIndexLayout extends Component {
     i18n.offChangeLocale(this.localeChanged.bind(this));
   }
 
+  /** 
+   * Return the count of application with specified status.
```

Comment:
if you gonna document functions like this (pretty good idea I think) you should do it for the `roundStats` one above too